packages/ui-react/package.json

```diff
@@ -45,7 +45,8 @@
     "typescript-plugin-css-modules": "^5.0.2",
     "vi-fetch": "^0.8.0",
     "vite-plugin-svgr": "^4.2.0",
-    "vitest": "^1.3.1"
+    "vitest": "^1.3.1",
+    "vitest-axe": "^1.0.0-pre.3"
```

Comment:
@langemike do you feel confident about this one?

Reply:
I sure am!